libs/CMakeLists.txt

```diff
@@ -9,11 +9,28 @@ include(HPX_Message)
 
 set(HPX_LIBS
   cache
+  collectives
   config
   preprocessor
   CACHE INTERNAL "" FORCE
 )
 
+if (HPX_WITH_TESTS_UNIT)
```

Comment:
`create_library_skeleton.py` should also be updated with this.

Reply:
And `tests.unit.modules` should probably be added as a dependency to `tests.unit` just like the header tests.